lib/fileSize.js

```diff
@@ -0,0 +1,92 @@
+import {formatNumber} from './number.js';
+import {getLanguage, validateFormatValue} from './common.js';
+
+export function getFileSizeDescriptor() {
+
+	const language = getLanguage();
+
+	const val = {
+		'gigabyte': 'GB',
+		'megabyte': 'MB',
+		'kilobyte': 'KB',
+		'bytes': 'bytes',
```

Comment:
This probably doesn't matter but just want to confirm this change is expected - it looks like previously for the `en` locales 'byte' and 'bytes' were actually capitalized (e.g., 'Bytes').

Reply:
Yep, very observant! I made that as a conscious change.